### DISCUSSION TITLE
How to check if those requests are duplicated?

### DISCUSSION THREAD

**Discussion by vnlevanduoc** [ThreeMammals/Ocelot#1767] [Q&A] [answered]
I'm using Ocelot, but I have a problem that I can't find a solution for
- My system is having a lot of requests sent at the same time
- How can I check if those requests are duplicated, can I block them
Requests submitted at the same time hh:mm:ss:sss

**Comment by ks1990cn** [comment]
Facing lot of hits and want to control that? RateLimiter is solution for it.

RateLimiter check `clientIdentity` internally and work on it.

Can you explain more on this, what exactly you are looking for if ratelimiter is not solution for you.

**Comment by raman-m** [comment, marked as answer]
Hi @vnlevanduoc !
For blocking purposes you can use current features like
- [Rate Limiting](https://ocelot.readthedocs.io/en/latest/features/ratelimiting.html) to deal with huge loads to a route, to allow exact number of requests per time period
- [Route Security Options](https://ocelot.readthedocs.io/en/latest/features/routing.html#security-options) to block/allow IP address ranges

There is no features to detect duplicates. But you can use Rate Limiter as an alternative.
